### DISCUSSION TITLE
Það eru svolítið margir búnir að fara á eftirlaun - 2024/5/6

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#266] [vefsida-ummæli]
# Það eru svolítið margir búnir að fara á eftirlaun - 2024/5/6

Við skulum ekki gleyma þessu aftur

https://www.biskupsreglan.com/frettir/eftirlaun

**Comment by TJ478** [comment]
Aldurinn er að sýna sig
